Update basic erc20/ether HTLC RFC (#138)
comit-network/RFCs: RFC-007-SWAP-Basic-Ether.adoc

|===
| Topic | keccack256(Topic)| Data  | Emitted When

| ascii to hex of `redeemed`
| 0x72656465656d6564000000000000000000000000000000000000000000000000
| `secret`
| The contract is redeemed

| ascii to hex of `refunded`
| 0x726566756e646564000000000000000000000000000000000000000000000000
| -
| The contract is refunded

|===

> The contract returns two different error values the failure case:

Reviewer: ```suggestion
The contract returns two different error values for the failure cases:
```

To be notified of the refund event, both parties SHOULD watch the blockchain for  `contract_address` emitting the `Refunded()` log.

=== Failure

A transaction to the HTLC will fail in two cases:

> . *If redeem was attempted with an invalid secret*: the contract will return `invalidSecret` int bytes (`0x696e76616c69645365637265740000000000000000000000000000000000000000`)

Reviewer: ```suggestion
. *If redeem was attempted with an invalid secret*: the contract will return `invalidSecret` in bytes (`0x696e76616c69645365637265740000000000000000000000000000000000000000`)
```

> . *If refund was attempted too early*: the contract will return `tooEarly` int bytes (`0x746f6f4561726c79000000000000000000000000000000000000000000000000`)

Reviewer: ```suggestion
. *If refund was attempted too early*: the contract will return `tooEarly` in bytes (`0x746f6f4561726c79000000000000000000000000000000000000000000000000`)
```
